app/assets/stylesheets/overrides.scss

```diff
@@ -118,6 +125,9 @@ height: 49px;
   color: $navmainfont;
   padding-left: 8px;
   padding-right: 8px;
+  :hover {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3